datastore/src/main/java/io/spine/server/storage/datastore/DsProperties.java

```diff
@@ -49,6 +60,22 @@ static void addAggregateIdProperty(Object aggregateId, Entity.Builder entity) {
         entity.set(aggregate_id.toString(), propertyValue);
     }
 
+    static void addCreatedProperty(Timestamp when, Entity.Builder entity) {
+        final long millis = Timestamps.toMillis(when);
+        final Date date = new Date(millis);
+        final DateTime dateTime = DateTime.copyFrom(date);
+        entity.set(AggregateEventRecordProperty.created.toString(), dateTime);
```

Comment:
Or, have an abstract method for `AggregateEventRecordProperty` and set values in impl.